Symfony/CS/Tests/Fixer/PSR2/ClassDefinitionFixerTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\PSR2;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @internal
+ */
+final class ClassDefinitionFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        return array(
+            array(
+                '<?php
+interface Test extends TestInterface8 implements /*a*/ /*b*/
+    TestInterface1, /* test */
+    TestInterface2, // test
+    TestInterface3, /**/
+    TestInterface4,
+    TestInterface5, /**/
+    TestInterface6
+{
+}',
+                '<?php
+interface Test
+extends
+    TestInterface8
+  implements  /*a*/    /*b*/TestInterface1   ,  /* test */
```

Comment:
:+1:  indeed
added (a failing) test as reminder to get this fixed
https://github.com/SpacePossum/PHP-CS-Fixer/commit/42c4c65e92e449e093b872b89f250d7ce937b15b